VERSION

```diff
@@ -0,0 +1 @@
+2.0.0
```

Comment:
OK, so is it good practice to have this hard coded in while we are developing? I know we are working towards this. Perhaps it should be something like 2.0.0beta1? Or even alpha1? I'm just thinking of the version guidelines shown here: https://setuptools.readthedocs.io/en/latest/setuptools.html. Ultimately if any of these pre-release versions get into a repository somewhere they should be listed behind the ultimate 2.0.0.

Reply:
Yeah, sounds good.

Reply:
From [here](https://gamedev.stackexchange.com/a/75814) and [here](https://blog.codinghorror.com/alpha-beta-and-sometimes-gamma/) I believe we fit into the pre-alpha development status.

Reply:
Defined to `2.0a0` based on this https://www.python.org/dev/peps/pep-0440/#id34